tests/figure.spec.ts

```diff
@@ -30,3 +39,44 @@ ${figureName} = Figure().plot_xy(series_name="Test", t=empty_table(100).update([
     page.locator('.iris-chart-panel .plotly.plot-container')
   ).toHaveScreenshot();
 });
+
+test('can set point shape and size', async () => {
+  const tableName = generateVarName();
+  const figureName = generateVarName();
+  const command = `from deephaven import empty_table
+from deephaven.plot import Figure
+import math
+
+math_funcs = ["sin", "cos"]
+shapes = ["SQUARE", "CIRCLE", "UP_TRIANGLE", "DOWN_TRIANGLE", "RIGHT_TRIANGLE", "LEFT_TRIANGLE", "DIAMOND"]
+# Unsupported shapes: ["ELLIPSE", "HORIZONTAL_RECTANGLE", "VERTICAL_RECTANGLE"]
+
+# Create a generic table that has enough columns to display all the shapes
+# Re-uses some of the funcs
+${tableName} = empty_table(50).update(["x=i*0.1"])
+for i in range(len(shapes)):
+    ${tableName} = ${tableName}.update([f"y{i}=Math.{math_funcs[i % len(math_funcs)]}(x+{math.floor(i / len(math_funcs))})"])
+
+# Generate the table and figure based on the shapes created
+${figureName} = Figure()
+for i in range(len(shapes)):
+    ${figureName} = ${figureName}.plot_xy(series_name=shapes[i], t=${tableName}, x="x", y=f"y{i}").point(shape=shapes[i], size=len(shapes) - i, visible=True)
+
+${figureName} = ${figureName}.show();`;
+
+  await typeInMonaco(page, command);
```

Comment:
Why `typeInMonaco` here instead of just pasting the command? I feel like we should only type in monaco when testing things directly related to typing (autocomplete, etc.). This feels fine to just paste which should also speed up the test a bit

Reply:
Yea, I couldn't get paste working right... let me try taking another crack at it.

Reply:
It might be the clipboard context isn't allowed in unsecure environments or something

You could try `page.howeverYouGetMonaco().fill(command)`

Reply:
Interestingly, `locator('.console-input textarea').fill(text)` does work in that it fills in the text immediately; _however_ is screws up the indenting in monaco editor:
![image](https://user-images.githubusercontent.com/4505624/210022572-fbf45f8e-6b02-4d66-8d07-2af6b041f12d.png)

I can simulate a paste event in Chromium and Safari though, and fall back to typing in Firefox (paste doesn't work there for some reason).

Reply:
For some reason on the latest run, the download table is flaking and had to run twice, with the first run taking ~2m. Not sure why it flaked out, but pasting here for posterity:
![image](https://user-images.githubusercontent.com/4505624/210023671-ea98b419-ab54-4a3d-bbb8-c6ad4f184b47.png)

As I wasn't changing this test, and it seems to be intermittent, I won't worry about it right now...